lambda_handlers/formatters/tests/test_json_formatter.py

```diff
@@ -41,6 +45,10 @@ class TestOutputFormatJSON:
     def test_valid(self, data_sample):
         assert json.dumps(data_sample) == output_format.json(data_sample)
 
+    def test_data_sample(self, data_sample):
+        content = json.dumps(data_sample)
```

Comment:
any objections making the content more explicit as in `'{"foo": "bar"}'`?

Reply:
do you mean to remove the data_sample fixture and write one test for each valid type?

Reply:
maybe something like this, less dynamic

Reply:
done?